Bossggle/schema.sql

```diff
@@ -0,0 +1,44 @@
+CREATE TABLE bossggle (
+       id INTEGER PRIMARY KEY,
```

Comment:
Your schema does not auto-increment the id. Therefore, your insert database commands need to have id values. Alternative is to put SERIAL on this line and you will not have to insert the id values for every insert command.